runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -166,10 +166,11 @@ private void setNativeContent(long newNativeContent) {
         assert mNativeContent == 0 && mCleanupReference == null && mContentViewCore == null;
 
         // Initialize ContentViewRenderView
-        boolean animated = XWalkPreferencesInternal.getValue(
+        boolean animated = animatable ? true : XWalkPreferencesInternal.getValue(
                 XWalkPreferencesInternal.ANIMATABLE_XWALK_VIEW);
```

Comment:
Do you mean:
```
boolean animated = animatable || XWalkPreferencesInternal.ANIMATABLE_XWALK_VIEW);
```

I thought we should prioritize the value of single XWalkView. Anyway, it can't be differentiated that whether the developer set the value to false or didn't set the value at all. So I think we should use Boolean object instead of primitive type.

At last, could you make it clear in the description that which is prioritized when the value of single XWalkView conflicts with global value.